src/main/java/no/unit/nva/cristin/projects/ProjectsWrapper.java

```diff
@@ -112,33 +115,42 @@ public void setHits(List<NvaProject> hits) {
      * Assigns value to id using supplied query parameters.
      *
      * @param queryParams the query params
-     * @return ProjectsWrapper object with id value from query parameters
+     * @return ProjectsWrapper object with id and firstRecord values from query parameters
      */
     public ProjectsWrapper usingQueryParams(Map<String, String> queryParams) {
         this.id = idUriFromParams(queryParams);
```

Comment:
At line 121 we calculate the query URI again (the other method being `CristinApiClient::generateQueryProjectsUrl`).
Ideally `CristinApiClient::queryProjects` should return an object containing the response and the query URI so that we know which query produced the response instead of calculating again. As a workaround now we could re-use the same method (`CristinApiClient::generateQueryProjectsUrl`), so that we do not implement the same logic twice.

Reply:
line 121 is the Nva URI while CristinApiClient::generateQueryProjectsUrl is Cristin URI